examples/vtk/file_viewer.py

```diff
@@ -0,0 +1,124 @@
+from trame.app import get_server
+from trame.ui.html import DivLayout
+from trame.widgets import html, client
+from trame_vtklocal.widgets import vtklocal
+from trame.decorators import TrameApp, change, trigger
+
+import argparse
```

Comment:
no need

Reply:
I use argparse below for `--filename`. I want it so that i can add more args in the future easily without implementing the logic myself.